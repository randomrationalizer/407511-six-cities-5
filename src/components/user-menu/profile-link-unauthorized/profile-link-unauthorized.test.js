import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import ProfileLinkUnauthorized from "./profile-link-unauthorized";


test(`ProfileLinkUnauthorized component renders correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <ProfileLinkUnauthorized />
        </BrowserRouter>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
