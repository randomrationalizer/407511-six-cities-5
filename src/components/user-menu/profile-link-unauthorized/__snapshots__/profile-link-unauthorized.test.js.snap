// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileLinkUnauthorized component renders correctly 1`] = `
<a
  className="header__nav-link header__nav-link--profile"
  href="/login"
  onClick={[Function]}
>
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />
  <span
    className="header__login"
  >
    Sign in
  </span>
</a>
`;
